sandbox/combined-tracks.html.example

```diff
@@ -25,7 +25,7 @@
     <pre>open http://localhost:9999/sandbox/index.html</pre>
   </div>
 
-  <video id="vid1" class="video-js vjs-default-skin" lang="en" controls preload="auto" width="640" height="264" poster="http://vjs.zencdn.net/v/oceans.png">
+  <video id="vid1" class="video-js vjs-default-skin" lang="en" controls preload="auto" width="640" height="264">
```

Comment:
Oh, I didn't know that the poster for ED was on the CDN. I can make that change - thanks @gkatsev!